.snyk

```diff
@@ -2,6 +2,10 @@
 version: v1.25.0
 ignore:
   SNYK-PYTHON-SETUPTOOLS-3180412:
+    - resources/python/requirements/3.7/requirements.txt
+    - resources/python/requirements/3.9/requirements.txt
+    - resources/python/requirements/3.10/requirements.txt
+    - resources/python/requirements/3.11/requirements.txt
```

Comment:
This does not sound like a good idea. Why this change?

Reply:
This only applies to ignore setuptools false positive ID `SNYK-PYTHON-SETUPTOOLS-3180412`.

Reply:
@julien-lang SG-31202

Reply:
Can you add this ticket ID in the YML as comment please?

Then, I can approve the PR.